### ISSUE TITLE
Finish this Integration with main website

### ISSUE THREAD

**Issue by ghost** [IIITians-Network/Map#1]
Check readme file.

Remaining Tasks

- Adding Data in store.js (JSON format)
- Changes in Frontend (if needed)
- Improving search function (JS)

@prashantkatiyar9777 @vibhanshujainiiitr please look at this.

**Comment by vibhanshujainiiitr** [comment]
Sure.
@prashantkatiyar9777 Please look into the process of integration with the main website.

I guess it would be great if we include this section on our home page, which @vimalverma558  had already done.
We just need to add the complete data, before going live on Home Page.

**Comment by IIITiansNetwork** [comment]
Yeah. I was in touch with Mr, Joshua. It was planned a long ago to include on the website.
Well, we have to edit the header of each webpage. Maybe that's why I am heading towards react. 

React developers are invited to join.

**Comment by IIITiansNetwork** [comment]
https://map.iiitiansnetwork.com/ Domain for the page. 
Deployed from 'deploy' branch.

**Comment by vimal-verma** [comment]
@IIITiansNetwork , I am already working on Integration with Home Page
this is demo website links - https://iiit-map.netlify.app/
this is Code(After successfully Integration, Pull request will be Opened ) -https://github.com/vimalverma558/Home-Page-Website

**Comment by vibhanshujainiiitr** [comment]
@vimalverma558 Exactly, it will be better if we add this directly on the home page.

**Comment by vimal-verma** [comment]
Integration Complete check this
IIITians-Network/Home-Page-Website#38
